Shared/Samples/Show geodesic sector and ellipse/ShowGeodesicSectorAndEllipseView.swift

```diff
@@ -0,0 +1,396 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowGeodesicSectorAndEllipseView: View {
+    /// The data model that helps determine the view.
+    @StateObject private var model = Model()
+    
+    /// Manages the presentation state of the menu.
+    @State private var isPresented: Bool = false
+    
+    var body: some View {
+        MapViewReader { mapView in
+            MapView(
+                map: model.map,
+                graphicsOverlays: model.graphicOverlays
+            )
+            .onSingleTapGesture { _, tapPoint in
+                model.center = tapPoint
+            }
+            .task(id: model.center) {
+                guard let center = model.center else { return }
+                await mapView.setViewpoint(
+                    Viewpoint(center: center, scale: 1e7)
+                )
+            }
+            .toolbar {
+                // The menu which holds the options that change the ellipse and sector.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Settings") {
+                        isPresented = true
+                    }
+                    .disabled(model.center == nil)
+                    .sheet(isPresented: $isPresented) {
+                        SectorSettingsView(model: model)
+                            .presentationDetents([.medium])
+                        Button("Close") {
+                            isPresented = false
+                        }
```

Comment:
Having a button at the bottom of the sheet does not fit standard UI conventions and is also visually unappealing (notice how the button is right up against the bottom of the sheet, with no padding?). Please follow my suggestion [here](https://github.com/Esri/arcgis-maps-sdk-swift-samples/pull/648#discussion_r2172151415). Also note the text for a button like this should be "Dismiss" instead of "Close".